spec/requests/api/v1/people_controller_index_spec.rb

```diff
@@ -31,6 +31,25 @@
 
         expect(People::Finder).to have_received(:new).with(police_national_computer: 'AB/1234567')
       end
+
+      context 'with audit' do
+        let(:audit) { ResponseAudit.last }
+        let!(:profile) { create(:profile, profile_identifiers: [{ identifier_type: 'police_national_computer', value: 'CD/7654321' }]) }
+        let(:person) { profile.person }
+        let(:params) { { filter: { police_national_computer: 'CD/7654321' } } }
+
+        before do
+          get '/api/v1/people', headers: headers, params: params
+        end
+
+        it 'creates an audit record' do
+          expect(audit).not_to be_nil
```

Comment:
I would avoid to put the expecting result inside a `let`, it make things more difficult to follow.

DRYing test is not something we should always aims.
The classic approach AAA "Arrange Act Assert" make things really simple to read.

```
response = ResponseAudit.last

expect(response).not_to be_nil
```
But I am aware that this is not the general approach we adopted for this project.
This is just to say that we should not overuse `let`.